src/components/Calculator.css

```diff
@@ -0,0 +1,45 @@
+.App {
+  width: 300px;
+  background-color: #f0f0f0;
+  margin-left: 64px;
+  margin-top: 98px;
+}
+
+.top {
+  font-size: 16px;
+  text-align: right;
+  margin-bottom: 10px;
+}
+
+.display {
+  font-size: 32px;
+  text-align: right;
+  margin-bottom: 20px;
+}
+
+.buttons {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-gap: 10px;
+}
+
+.Button {
+  padding: 10px;
+  font-size: 20px;
+  text-align: center;
+  background-color: #fff;
+  border: 1px solid #ccc;
+  cursor: pointer;
+}
+
+.function {
+  background-color: #f0f0f0;
+}
+
+.operator {
+  background-color: #d0d0d0;
+}
+
+.zero {
+  grid-column: span 2;
+}
```

Comment:
- [ ] As stated in the project the` Calculator.js` component design should match the following given design:So kindly try to adjust background color of the  the calculator just the like the Design
- Your Design Look likes this :- 
![Cal](https://github.com/kayjeee/Micro_Verse_React_Maths_Mathematician_App_v2/assets/93598189/a10e9a74-7593-4d88-aeb1-ed9dc75aaa7e)

- The Given Design Look likes this 

![Desn](https://github.com/kayjeee/Micro_Verse_React_Maths_Mathematician_App_v2/assets/93598189/3ee8a131-685f-4175-92c6-84b60bc90dfa)

Reply:
Dear @Abenezer-Tilahun 

Thank you for your feedback on the pull request. I have made the requested adjustment to the background color of the calculator component to match the provided design. 🎨🔧 It now closely resembles the desired look.

I would like to kindly remind you to follow my GitHub account ([Kayjee](https://github.com/kayjeee)) so that we can stay connected and support each other's work. 🌟🤝 If you find the project impressive and it meets your expectations, I would greatly appreciate it if you could show your support by starring ⭐️ the project ([Micro_Verse_React_Maths_Mathematician_App_v2](https://github.com/kayjeee/Micro_Verse_React_Maths_Mathematician_App_v2)). Your encouragement and endorsement mean a lot to me and would greatly contribute to my professional journey.

Once again, thank you for your time and consideration. I look forward to any further feedback or suggestions you may have.

Best regards,
@kayjeee